src/org/ods/OdsPipeline.groovy

```diff
@@ -18,12 +18,15 @@ class OdsPipeline implements Serializable {
   // Main entry point.
   def execute(Closure stages) {
     logger.info "***** Starting ODS Pipeline *****"
+    checkForMultiRepoBuild()
```

Comment:
A `checkFor` function that doesn't return something has a smell. The function doesn't only check, but sets configuration properties, right? Please rename this to something more appropriate then. Suggestion:`prepareFor...`?